v2g-liberty-cards/src/edit-charger-settings-dialog.ts

```diff
@@ -296,23 +297,33 @@ class EditChargerSettingsDialog extends DialogBase {
     this.closeDialog();
   }
 
-  static styles = css`
-    .invalid {
-      color: var(--error-color);
-    }
 
-    .name {
-      font-weight: bold;
-    }
 
-    .success ha-svg-icon {
-      color: var(--success-color);
-      padding-right: 2rem;
-    }
+  static styles = [
+    styles,
+    css`
+      .invalid {
+        color: var(--error-color);
+      }
+
+      .name {
+        font-weight: bold;
+      }
+
+      // All of these don't seem to reach the hr element...
+      // hr,
+      // ha-markdown::part(hr),
```

Comment:
It seems like a bad idea to me to want to style using css, instead of using markdown anyway ...

Reply:
Yes and no. The markdown `---` creates a <hr/> element that i would like to style more subtle than it is default.

Reply:
But leaving it for now as i don't get it to work.